tests/core/test_typing.py

```diff
@@ -6,6 +6,8 @@
 import logging
 import typing as t
 
+import sqlalchemy
```

Comment:
```suggestion
import pytest
import sqlalchemy
```

and you'll also need to remove the `if t.TYPE_CHECKING:` block